doc/tsc-meetings/2015-06-03.md

```diff
@@ -0,0 +1,166 @@
+# Node.js Foundation TSC Meeting 2015-06-03
+
+## Links
+
+* **Audio Recording**: https://soundcloud.com/node-foundation/tsc-meeting-2015-06-03
+* **GitHub Issue**: https://github.com/nodejs/node-convergence-archive/issues/47
+* **Original Minutes Google Doc**: https://docs.google.com/document/d/1sTD0uryasBR15UBzEbJj3oHYtnuN9ZIqVxA2A_-N56E
+
+## Agenda
+
+Extracted from **tsc-agenda** labelled issues and pull requests prior to meeting.
+
+### nodejs/io.js
+
+* Add working group state per Project Lifecycle. [#1880](https://github.com/nodejs/io.js/pull/1880)
+* Proposal: Split TSC agenda in to two meetings [#1879](https://github.com/nodejs/io.js/issues/1879)
+* Chrome 43 released; time for V8 4.3! [#1735](https://github.com/nodejs/io.js/issues/1735)
+* TSC needs to elect a board representative. [#1697](https://github.com/nodejs/io.js/issues/1697)
+* Expose `deepEqual` and `deepStrictEqual` in `util`. #1172 [#1177](https://github.com/nodejs/io.js/pull/1177)
+
+## Present
+
+* Rod
+* Mikeal
+* Shigeki
+* Chris
+* Colin
+* Julien
+* James
+* Michael
+* Bert
+* Fedor
+* Jeremiah
+* Domenic
+* Alexis
+* Ben
+
+## Quick stand-up
+
+* Rod: working on the build, looking into npm smoke-testing
+* Mikeal: getting foundation legalities and officialities in order
+* Shigeki: working on security issues, reviewing root cert update
+* Chris: working on getting npm static analysis working [estoc](https://github.com/chrisdickinson/estoc)
+* Colin: reviewing prs and issues, doing some libuv work (uv_os_homedir, which landed in libuv 1.6.0)
+* Julien: vactaion, nodejs.org downtime postmortem
+* James: working on convergence, triaging joyent/node issues
+* Michael: triaging some joyent/node issues, working on powerpc build and npm testing
+* Bert: not much, looked at some issues, discussing with Saul if libuv should move into the Node Foundation
+* Fedor: reviewed some pull requests and issues, working on some openssl mode
+* Jeremiah: issues, convergence, _unrefActive timers work, looking at improving Ben’s binary heap implementation
```

Comment:
Maybe put `_unrefActive` in backticks, GH-flavored markdown is interpreting it as italics.